ob-go.el

```diff
@@ -62,19 +62,26 @@
 ;;   package declaration.
 
 ;;; Code:
-(require 'go-mode)
 (require 'org)
 (require 'ob)
 (require 'ob-eval)
 (require 'ob-ref)
 
+(defun ob-go-get-var (params)
+  "org-babel-get-header was removed in org version 8.3.3"
+  (let* ((fversion (org-version))
+        (version (string-to-int fversion)))
+    (if (< version 8.3)
+        (mapcar #'cdr (org-babel-get-header params :var))
+      (org-babel--get-vars params))))
+
 
 ;; optionally define a file extension for this language
 (add-to-list 'org-babel-tangle-lang-exts '("go" . "go"))
 
 (defvar org-babel-default-header-args:go '())
 
-(defvar org-babel-go-command "go"
+(defvar org-babel-go-compiler "go"
```

Comment:
Right. 

I'd made the change because ob-go fit perfectly with ob-C code, and maybe make sense to it be integrated into ob-C code base in the future.